src/feasibility_checker.jl

```diff
@@ -0,0 +1,161 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+using LinearAlgebra
+
+"""
+    primal_feasibility_report(
+        model::Model,
+        point::AbstractDict{VariableRef,Float64};
```

Comment:
I would have a default value for `point` equal to the primal value. This is want people will need 99% of the time and it will be annoying to construct this dict (people could even construct it incorrectly and get errors).